sdk/core/Azure.Core.Experimental/src/Amqp/AmqpDataBody.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.Core.Amqp
+{
+    /// <summary>
+    /// Represents the data body of an AMQP message.
+    /// This consists of one or more data sections.
+    /// http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-data
```

Comment:
nit:  I'd recommend moving this to `<seealso href="..." />` for consistency with other docs using external references.